api/IDBVersionChangeEvent.json

```diff
@@ -294,7 +294,7 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/IDBVersionChangeEvent/version",
           "support": {
             "chrome": {
-              "version_added": "12"
+              "version_added": "24"
```

Comment:
Here's the CL for this feature: https://chromium.googlesource.com/chromium/src/+/9612cdfd1a438fc2b276cb5f71e5d85bffedcd67

It's too old for our version look up system to contain its commit hash, but the time frame right for version 12. It was definitely not 24.